Books/app/routes.js

```diff
@@ -1,37 +1,38 @@
-var path = require('path');
-var Book = require('./model');
-var routes = function(app) {
-    app.get('/book', function(req, res) {
-        Book.find({}, function(err, result) {
-            if ( err ) throw err;
+const path = require('path');
+const Book = require('./model');
+
+const routes = (app) => {
+    app.get('/book', (req, res) => {
+        Book.find({}, (err, result) => {
+            if (err) throw err;
             res.json(result);
         });
     });
-    app.post('/book', function(req, res) {
-        var book = new Book( {
-            name:req.body.name,
-            isbn:req.body.isbn,
-            author:req.body.author,
-            pages:req.body.pages
+    app.post('/book', (req, res) => {
+        const book = new Book({
```

Comment:
I personally find destructuring less readable. Do we happen to have a JS style guide for Microsoft that we can check?

Reply:
That's a fair point. I just ran into this myself on something else and wasn't sure that it was more readable, but perhaps it's only because I'm not used to seeing it.

On the style guide - no and we need one. Started a thread to try and get some more insight into that. Let's park this for now. I'll close it.